modules/microshift-custom-ca-proc.adoc

```diff
@@ -0,0 +1,227 @@
+----
++
+.Example output
+[source,terminal]
+----
+oc get node
+NAME                             STATUS   ROLES                         AGE   VERSION
+dhcp-1-235-195.arm.example.com   Ready    control-plane,master,worker   76m   v1.29.2
+----
+
+. Check that the `openshift-service-ca` is running by entering the following command:
++
+[source,terminal]
+----
+$ oc --insecure-skip-tls-verify get pod -A
+----
++
+.Example output
+[source,terminal]
+----
+oc get pod -A
+NAMESPACE                              NAME                                       READY   STATUS    RESTARTS      AGE
+kube-system                            csi-snapshot-controller-5bdbd484f4-c6prb   1/1     Running   0             22m
+kube-system                            csi-snapshot-webhook-957f6d687-vb7fl       1/1     Running   0             22m
+openshift-dns                          dns-default-gbkx8                          2/2     Running   0             22m
+openshift-dns                          node-resolver-lqd7d                        1/1     Running   0             22m
+openshift-ingress                      router-default-784fb9bf84-khfsh            1/1     Running   0             22m
+openshift-operator-lifecycle-manager   catalog-operator-645c787db7-6vvst          1/1     Running   0             22m
+openshift-operator-lifecycle-manager   olm-operator-b75ccb977-dh4wx               1/1     Running   0             22m
+openshift-ovn-kubernetes               ovnkube-master-gr8xc                       4/4     Running   1 (22m ago)   22m
+openshift-ovn-kubernetes               ovnkube-node-r9gth                         1/1     Running   1 (22m ago)   22m
+openshift-service-ca                   service-ca-85fc4d5bb7-5g9vp                1/1     Running   0             22m
+openshift-storage                      topolvm-controller-b94d5885b-7dmms         5/5     Running   0             22m
+openshift-storage                      topolvm-node-x5hgs                         4/4     Running   0             22m
+----
++
+--
+[NOTE]
+====
+If you use the `oc get node` command, the error `Unable to connect to the server: tls: failed to verify certificate: x509: certificate signed by unknown authority` is expected. Using `--insecure-skip-tls-verify` resolves the error.
+====
+--
+
+. Add the new CA file to $KUBECONFIG by running the following command:
++
+[source,terminal]
+----
+cat /tmp/caCert.pem /tmp/certificate-authority-data-origin.crt > /tmp/certificate-authority-data-new.crt
+sed -i "s/certificate-authority-data: .*/certificate-authority-data: $(base64 -w0 /tmp/certificate-authority-data-new.crt)/" $KUBECONFIG
+----
+
+. Confirm that the new CA has replaced the previous one by using the following steps:
+.. After customizing the new CA, confirm that original certificate authority does not work by running the following command:
++
+[source,termina]
+----
+$ curl --cacert /tmp/certificate-authority-data-origin.crt https://${fqdn_name}:6443/healthz
```

Comment:
in the previous step we changed the generated kubeconfig file content  to include the custom CA,
so it has not effect on the verification `curl` commands.